performance_curves/performance_curve_bundle.py

```diff
@@ -0,0 +1,54 @@
+from performance_curves.performance_curve import *
+
+
+class PerformanceCurveBundle:
+    def __init__(self,
+                 y_true: np.ndarray,
+                 y_scores: List[np.ndarray],
+                 metric: Metric,
+                 model_names: List[str],
```

Comment:
If we expect a 1:1 relationship between `y_scores` and `model_name` then we should consider asking for a `Mapping[src, np.ndarray]`. Unless order matters, in which case we can think of something else.

Reply:
Good point! Both of these arguments are now condensed into one, named `method_scores` of type `Mapping[str, np.ndarray]`. I named them `method_` instead of `model_` following your first suggestion above.